roles/create_client_library_conf/templates/conf.json.j2

```diff
@@ -1,6 +1,9 @@
 {
     "version": 0.1,
     "init_time": {{ client_library_init_time }},
+    {% if gen2 | bool %}
```

Comment:
Extremely minor comment - can you move the jinja statements so it's indented the same as the other lines in the template?